snapshots/input/syntax/src/import.ts

```diff
@@ -1,3 +1,4 @@
+import * as ts from 'typescript'
```

Comment:
nit: Could you define a namespace in a different file (similar to `./class` and `./enum` below) and use that here? We try to make tests as self-contained as possible, whereas using `SyntaxKind` and `ArrayType` kinda' goes against that. If one is looking at the test by itself, it isn't obvious whether the `'typescript'`/`ts` is load-bearing or if it is more general.